src/app/components/wrapper-components/GameWrapper.ts

```diff
@@ -127,11 +128,10 @@ export abstract class GameWrapper<P extends Comparable> {
         const LOCAL_VERBOSE: boolean = false;
         display(GameWrapper.VERBOSE || LOCAL_VERBOSE,
                 { gameWrapper_receiveValidMove_AKA_chooseMove: { move, state, scores } });
-        if (this.isPlayerTurn() === false) {
-            return MGPValidation.failure(GameWrapperMessages.NOT_YOUR_TURN());
-        }
-        if (this.endGame) {
-            return MGPValidation.failure($localize`The game has ended.`);
+        const userPlayValidity: MGPValidation = this.canUserPlay('none');
```

Comment:
seul le gameComponent peut triggerer receiveValidMove, comment-ce fait-ce qu'il permet, si canUserPlay()===false, qu'il appelle receiveValidMove ?

Reply:
Bon point !
Investigation: je désactive le check et roule les tests
LocalGameWrapperComponent Using AI should reject human move if it tries to play (without click) when it is not its turn FAILED

Je fix donc le test qui était une relique du passée appelant receiveValidMove !

Reply:
... "if it tries to play (without click)" COMMENT IL FAIT L'HUMAIN POUR JOUER SANS CLICK :D ?

Reply:
LOL ben il fait pas ! donc enlevé le "(without click)", mais test gardé car utile !

Reply:
Non mais utile à tester quel cas ?

Reply:
que quand t'essaye de cliquer et que c'est pas ton tour, ça te le dit !